docs/configuration-reference.md

```diff
@@ -1,16 +1,74 @@
 # Configuration reference
+**Quick Links:**
```

Comment:
I thought to create this Quick Link section to make it easier to navigate the long config-reference page. But since readthedocs.io creates a menu from the headings this bit might be pointless. What do you think?

Reply:
I think it's a good idea.